src/FileInfo.cpp

```diff
@@ -66,15 +66,15 @@ FileInfo GetFileInfo(const char *path)
 
     if (strlen(path) >= MAX_PATH)
```

Comment:
This is not safe, from what you've told me - a relative path that is less than MAX_PATH chars could expand to a path that is too long once we pass it to the WinAPI.